CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 - [Previous Changelogs](https://github.com/eclipse-theia/theia/tree/master/doc/changelogs/)
 
+
+## v.1.27.0
+- [console] fixed issue in Debug console where console history was not being trimmed in accordance with the maximum commands limit [#10598](https://github.com/eclipse-theia/theia/pull/10598)
```

Comment:
The next release would be [`1.26.0`](https://github.com/eclipse-theia/theia/releases) not `1.27.0` so we can move the item down to `1.26.0`.

Reply:
Ok, this is updated in the next commits